Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -123,7 +123,7 @@ def __init__(self):
         self.preprocessors = None
         self.outdated_settings = False
         self.setup_layout()
-        QTimer.singleShot(0, self.apply)
+        QTimer.singleShot(0, self.update_learner)
```

Comment:
@ales-erjavec, @astaric, please see #1601, which should also fix this problem. Two other widgets in education remove buttons, while (AFAIK) no widgets in core, bio, datafusion, network, prototypes or text extend `BaseLearner`.

What about replacing the above line with
`QTimer.singleShot(0, getattr(self, "unconditional_apply", self.apply))`?

If the widget does not have `unconditional_apply` it has either left `apply` as it is, hence calling it has the same effect as `unconditional_apply` would, or it has overridden `apply`, so we must call it anyway. If the widget creates its own Apply button with a callback with a different name, then it will have to reimplement the output initialization, too.